packages/pkl-ts-parser/scripts/download.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! -f "package.json" ]; then
+  echo "Please run this script from the root of the project"
+  exit 1
+fi
+
+pushd src/antlr4
+
+wget https://raw.githubusercontent.com/apple/pkl/0.25.2/pkl-core/src/main/antlr/PklLexer.g4 -c
```

Comment:
Can we make these two urls and pass them to script or read them from .env file?
I am not sure